zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -446,6 +455,9 @@ impl<F: Field> ExecutionGadget<F> for CallOpGadget<F> {
             step.rw_indices[5],
         ]
         .map(|idx| block.rws[idx].call_context_value());
+        let error_depth = depth.low_u64() > 1024;
+        self.is_depth_ok
+            .assign(region, offset, F::from(depth.low_u64()), F::from(1025))?;
```

Comment:
change error_depth to is_error_depth to be more consistent with other names like is_insufficient?